File: app/src/main/java/com/BlizzardArmory/diablo/Item/Attributes.java
package com.BlizzardArmory.diablo.Item;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class Attributes {


    @SerializedName("other")
    @Expose
    private List<Object> other;
    @SerializedName("primary")
    @Expose
    private List<Primary> primary;
    @SerializedName("secondary")
    @Expose
    private List<Secondary> secondary;

    public List<Object> getOther() {
        return other;
    }

    public List<Primary> getPrimary() {
        return primary;
    }

    public List<Secondary> getSecondary() {
        return secondary;
    }

}

File: app/src/main/java/com/BlizzardArmory/model/diablo/diablo3/item/Attributes.kt
package com.BlizzardArmory.model.diablo.diablo3.item

import androidx.annotation.Keep
import com.google.gson.annotations.SerializedName


/**
 * The type Attributes.
 */
@Keep
data class Attributes(

    @SerializedName("other")

    val other: List<Any>,

    @SerializedName("primary")

    val primary: List<Primary>,

    @SerializedName("secondary")

    val secondary: List<Secondary>
)